extensions/amp-analytics/amp-analytics.md

```diff
@@ -291,6 +291,39 @@ In this example, all requests are valid.
 
 Some analytics providers have an already-provided configuration, which you use via the `type` attribute. If you are using an analytics provider, you may not need to include requests information. See your vendor documentation to find out if requests need to be configured, and how.
 
+##### Request Origin
```

Comment:
nit: Could you please add `origin` support here https://github.com/ampproject/amphtml/pull/23138/files#diff-5d3a07ba74f4bc164e1416d74355aa8aR274
And link to this section.